Export/Scripts/bases.lua

```diff
@@ -164,7 +164,11 @@ directiveTable.base = function(state, args, out)
 end
 
 directiveTable.baseMatch = function(state, args, out)
-	for i, baseItemType in ipairs(dat("BaseItemTypes"):GetRowList("Id", args, true)) do
+	local key = "Id"
+	if args:match("Abstract") then
```

Comment:
This is most definitely a hack, it might be better to change every matcher to look at the BaseType column instead of Id, but then I'd have to change all of the .txt files and it'd be a bigger change than this PR was intended for.  Definitely better for future proofing, though.  I think some special logic would be needed for flasks, but that might be it

Reply:
Ok, I found a way to make it better without having to change too much